smithy-cli/src/test/resources/software/amazon/smithy/cli/commands/smithy-lock-test.json

```
@@ -0,0 +1,11 @@
+{
+  "_AUTOGENERATED_FILE_DO_NOT_MODIFY_MANUALLY_":"",
+  "version":"1.0",
+  "config_hash":-1856284556,
```

We use camelCase everywhere else, so we should use it here too.
```suggestion
  "configHash":-1856284556,
```